payments/swish/direct.md

```diff
@@ -0,0 +1,297 @@
+|             | └─➔&nbsp;`ecomOnlyEnabled`             | `boolean`  | `true` if to only enable Swish on web based transactions.; otherwise `false` to also enable Swish transactions via in-app payments.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+
+{:.code-header}
+**Response**
+
+```http
+POST /psp/swish/payments HTTP/1.1
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "payment": {
+        "id": "/psp/swish/payments/{{ page.paymentId }}",
+        "number": 992308,
+        "created": "2017-10-23T08:38:57.2248733Z",
+        "instrument": "Swish",
+        "operation": "Purchase",
+        "intent": "Sale",
+        "state": "Ready",
+        "currency": "SEK",
+        "amount": 0,
+        "description": "Test Purchase",
+        "payerReference": "AB1234",
+        "initiatingSystemUserAgent": "Mozilla/5.0",
+        "userAgent": "Mozilla/5.0...",
+        "language": "sv-SE",
+        "urls": {
+            "id": "/psp/swish/payments/{{ page.paymentId }}/urls"
+        },
+        "payeeInfo": {
+            "id": "/psp/swish/payments/{{ page.paymentId }}/payeeinfo"
+        }
+    },
+    "operations": [
+        {
+            "method": "PATCH",
+            "href": "http://{{ page.apiUrl }}/psp/swish/payments/20dfbcb9-587a-4ce9-e63e-08d519f1802f",
+            "rel": "update-payment-abort"
+        },
+        {
+            "method": "POST",
+            "href": "http://{{ page.apiUrl }}/psp/swish/payments/20dfbcb9-587a-4ce9-e63e-08d519f1802f/sales",
+            "rel": "create-sale"
+        }
+    ]
+}
+```
+
+## Create Sale transaction
+
+This operation creates a sales transaction in the direct payment scenario.
+This is managed either by sending a `POST` request as seen
+below, or by directing the end-user to the hosted payment pages. Note that the
+`msisdn` value (the consumer/end-user's mobile number) is required in this
+request.
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/swish/payments/<paymentId>/sales HTTP/1.1
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "transaction": {
+        "msisdn": "+46739000001"
+    }
+}
+
+```
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "payment": "/psp/swish/payments/20dfbcb9-587a-4ce9-e63e-08d519f1802f",
+    "sale": {
+        "date": "23.10.2017 08:39:37 +00:00",
```

Comment:
Is this `date` correct? The format looks very, very wrong. If this is the date format our Swish API outputs, it's a major bug.